### PR TITLE
feature(apple-store):

### PR DESCRIPTION
**Pull request by mujohn26**
#### What does this PR do?
- worked on apple store webpage

#### Description of Task to be completed?
- added homepage photos and backgrounds
- added homepage styles and fonts
- Added  README file
#### How should this be manually tested?
- clone the repository
- open index.html in your browser
#### Any background context you want to provide?
- N/A

#### Screenshot
Screen Shot 2020-06-24 at 15 25 53

#### Questions:
- N/A

### CODE REVIEW

**Review by jofer86 — CHANGES_REQUESTED** [review]
## :wave: :wave: Hello!, I'm Jorge, and  I'm here to review this Pull Request, let's take a look shall we :eye: :eye: :eye: 
- Changes marked as **[OPTIONAL]** are not hard requirements but are nice to have.
- The first thing that comes to my attention is the way the site is displayed on a 1080p screen 
image
See there are some discrepancies here with the way your page looks like there is a small margin in all boarders, and the project should be the entire screen of your project, please do tend to that.
- This other part of your component also needs some tweaking : 
image
When the original one looks like this: 
image
You need to give your component border radius property to soften the edges.
- Also regarding that component, there are elements missing you should  be showing 10 elements, text should be dead center in every part.
- There is also this effect the navbar has 
bar
- We also need to improve the typography to be the same or at least the same style as the original, by style I mean that it should be a sans serif font, if you are in doubt to what that means google it for more reference, since you should use a sans serif font to relate more to the original one.
- Make sure the proportois stay as true to the original design as possible, keeping the ratio of the images is also necesary to achieve this.
- Let's make sure the we use anchor tags so that we can get the link feel on the mouse, or feel free to add it as a pseudo selector in your css.
--------------------------------------------------------------------
Cheers, 
Jorge Fernando Rincón Figueroa.
Technical Support Engineer.
**Slack: @jofer86**

**Review by sumancrest0001 — COMMENTED** [review]
**Status: changes required
Nice work  @mujohn26,** 
You have made most of the changes requested by the TSE and the project is shaping nicely. But there are a couple of issues that are not solved from the previous review. Let's fix all those minor issues before moving ahead.
[REQUIRED] Let's fix the navbar issues addressed by the previous TSE.

File: index.html
Comment (sumancrest0001) [inline index.html]: [REQUIRED] These four articles should have links as in the original site.[orignal page](https://web.archive.org/web/20140301004610/http://www.apple.com/)
[REQUIRED] Let's fix the HTML validator errors. I am using [this validator](https://validator.w3.org/#validate_by_upload)

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
[Optional] It seems like you forgot to add the US logo at the right side footer. I request you to add to give it a  similar look.

**Kindly make the requested changes and fill up another review form.**
If you have any queries regarding the review or need help, contact me on slack **Suman Shrestha**.

**Happy coding**

File: index.html
Comment (mujohn26) [reply index.html]: Resolved

**Comment by mujohn26** [comment]
> ## 👋 👋 Hello!, I'm Jorge, and I'm here to review this Pull Request, let's take a look shall we 👁️ 👁️ 👁️
> * Changes marked as **[OPTIONAL]** are not hard requirements but are nice to have.
> * The first thing that comes to my attention is the way the site is displayed on a 1080p screen
>   image
>   See there are some discrepancies here with the way your page looks like there is a small margin in all boarders, and the project should be the entire screen of your project, please do tend to that.
> * This other part of your component also needs some tweaking :
>   image
>   When the original one looks like this:
>   image
>   You need to give your component border radius property to soften the edges.
> * Also regarding that component, there are elements missing you should  be showing 10 elements, text should be dead center in every part.
> * There is also this effect the navbar has
>   bar
> * We also need to improve the typography to be the same or at least the same style as the original, by style I mean that it should be a sans serif font, if you are in doubt to what that means google it for more reference, since you should use a sans serif font to relate more to the original one.
> * Make sure the proportois stay as true to the original design as possible, keeping the ratio of the images is also necesary to achieve this.
> * Let's make sure the we use anchor tags so that we can get the link feel on the mouse, or feel free to add it as a pseudo selector in your css.
> 
> Cheers,
> Jorge Fernando Rincón Figueroa.
> Technical Support Engineer.
> **Slack: @jofer86**

I resolved the feedback

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
### Hello @mujohn26 :wave: I am Alexandru, your code reviewer for this project
I can see you implemented the changes required by @sumancrest0001 (TSE) gret job on that :clap:, you are still missing some important fixes from @jofer86 (TSE).
### :stop_sign: Required Changes :construction: :construction_worker: 
> * The first thing that comes to my attention is the way the site is displayed on a 1080p screen
>   image
>   See there are some discrepancies here with the way your page looks like there is a small margin in all boarders, and the project should be the entire screen of your project, please do tend to that.
> * This other part of your component also needs some tweaking :
>   image
>   When the original one looks like this:
>   image
>   You need to give your component border radius property to soften the edges.
> ~*Also regarding that component, there are elements missing you should  be showing 10 elements~ :heavy_check_mark:  
- [ ] **In the navbar the text should be dead center in every part,  yours still looks messy**
> ~* There is also this effect the navbar has
>   bar~ :heavy_check_mark: 
> * We also need to improve the typography to be the same or at least the same style as the original, by style I mean that it should be a sans serif font, if you are in doubt to what that means google it for more reference, since you should use a sans serif font to relate more to the original one.
- [ ] **You applied the required typography  but only to the navbar, please apply the `sans serif` font globally so that the font style will be the same everywhere**
> * Make sure the proportois stay as true to the original design as possible, keeping the ratio of the images is also necesary to achieve this.
> ~* Let's make sure the we use anchor tags so that we can get the link feel on the mouse, or feel free to add it as a pseudo selector in your css.~ :heavy_check_mark: 

### <em>Kindly fix the REQUIRED issues and send another code review request.

Cheers and Happy coding! :clap: 

Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>
### :stop_sign: Status: Required Changes

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: :stop_sign: 

**HELLO** @mujohn26  :wave:  **I AM PETER ROBERT A TSE**
I have reviewed your project and good work improving on most of the required changes :clap: 

However, there are a few remaining issues that have prevented me from approving your project. Please improve on the following: :smile: 

- [x]  There is an error in your HTML. Kindly use [this tool](https://validator.w3.org/) to validate your HTML and fix all errors before resubmitting
validator error

- [x] Still, there is a problem with your navigation bar. For this project to be approved you need to fix that just like suggested by the previous TSE. 
menu bar

**NB**  Kindly make it look like the [original](https://web.archive.org/web/20140301004610/http://www.apple.com/)

- [x] I have also noticed that there is some space on the left, right, and top of your webpage kindly remove them.
header image fill

**Once your through submit your project for another review** :smile: 

### HAPPY CODING  :computer:

File: index.html
Comment (HADDADSOHAIB) [inline index.html]: Inline styles are a bad practice, move it to css file

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status required changes :recycle: 

### Hi everyone :handshake:

### The project is looking great good job. But still, some changes that you have to handle before we could approve the project:
 - As required by the previous TSE, there some validators issues:
Screenshot from 2020-06-24 12-04-49
- You have inline style in your HTML, you should move that to your css file. Check my comments for more informations.
- There is still some work to be done on your navigation, please try to center the navigation elements in the center.
Screenshot from 2020-06-24 12-11-02
original:
Screenshot from 2020-06-24 12-16-02
- as required by the previous TSE, please try to handle this issue:
Screenshot from 2020-06-24 12-16-59

### Once done with these changes, ask for another code review

**Comment by mujohn26** [comment]
> ### Status required changes ♻️
> ### Hi everyone 🤝
> ### The project is looking great good job. But still, some changes that you have to handle before we could approve the project:
> * As required by the previous TSE, there some validators issues:
>   Screenshot from 2020-06-24 12-04-49
> * You have inline style in your HTML, you should move that to your css file. Check my comments for more informations.
> * There is still some work to be done on your navigation, please try to center the navigation elements in the center.
>   Screenshot from 2020-06-24 12-11-02
>   original:
>   Screenshot from 2020-06-24 12-16-02
> * as required by the previous TSE, please try to handle this issue:
>   Screenshot from 2020-06-24 12-16-59
> 
> ### Once done with these changes, ask for another code review

Hello @HADDADSOHAIB I think you have no pulled the changes on the PR because I already fixed that problem of navbar and space

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @mujohn26  :wave:
-
You have done a good job with the changes! :tada: Keep it up.
We, however, do have some few changes to carry out before we proceed (see review comment).
##
- Before we merge this, we need to ensure nothing is messed up. Just as @jofer86 mentioned earlier, while the changes are visible for the last review, there are still some areas that need tweaking on large screens (desktop view). Please set the difference below 👇 and work the suggested changes then submit for another code review. Thank you.

|Source Page|Clone Page|
|:---:|:---:|
|source|clone|

- The search area is still not properly centred as with the source page. Kindly work on that.
-  The four elements should uniformly cover the entire width of the bottom section of the content as seen in the source page. Please fix this.
-  The footer links are also not in their rightful place. Kindly work on them as well. 
- As for the navbar, you might want to increase it's margin-top a bit. 
- Good job with the Stylesheets. There are no errors reported but please make sure you don't repeat yourself much often. Aim to adhere to the DRY principle. Check this [article ](https://speckyboy.com/good-bad-css-practices) for support to help you with some of the rules you might want to pay attention to. It is a  [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright.
##
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***


Hello Christian it's working perfectly on my side the alignments and also search is centered correcrtly you can check on this screenshot 
Screen Shot 2020-06-24 at 15 25 53

**Review by christianotieno — CHANGES_REQUESTED** [review]
Hello John.. Kindly let's connect on slack, please.

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @mujohn26, great job on this project :+1: All the comments have been addressed, but there is a horizontal scroll that needs to be fixed before the project can be approved :slightly_smiling_face: 

The screenshot below will help us understand the issue.
mfix
We could see that the problem arises from the four columns, so a quick fix would be to set the container wrapping the four columns to width: 100%, then each of the four columns can have a width:25%. By doing this we would not think about another person's screen size because it is percentage.

Kindly makes this change and submit another code review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](linkedin.com/in/daniel-larbi-addo/)

**Review by trekab — CHANGES_REQUESTED** [review]
## Changes Required! :recycle: 
Hi @mujohn26! First of all, I'd like to commend you for the hard work you are putting to meet this project's requirements. There are some issues I'd like to bring to your attention though. These include:
- [ ] You have one commit for all the changes you have ever made on this branch. I have noticed that you are using [force pushing](https://www.atlassian.com/git/tutorials/syncing/git-push#:~:text=Force%20Pushing,one%2C%20then%20try%20pushing%20again.) for any new changes you make. This is a bad practice. Your commit history is important for any project you work on. __Please avoid this__.
- [ ] Files not related to the project should not be committed and added to .gitignore (e.g. `.DS_Store`) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- [ ] Your clone appears broken. Check out the screenshot for highlighted areas of concern.
Screenshot
   - Your navbar is not centered in the middle of the screen. You gave it a fixed width and a left margin of 25%. This will surely make it appear differently on different screen sizes. 
   - You have a bottom scroll bar.

- [ ] __[OPTIONAL]__ Use the [`<nav>` tag](http://html5doctor.com/nav-element/) for the major navigation links. I have noticed your use of the `divs` for your `navbar`. This is not the best practice.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

Hello @kabareebe I have fixed the feedbacks you gave me

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle: 

### Hi everyone :hand:

### good job, you implemented a lot of the required changes, but somehow other things get broke, this is a screen shot of your project when I open it:
Screenshot from 2020-06-25 12-28-35
- For the huge button:
Screenshot from 2020-06-25 12-34-55
it can be fixed by adding this to th css:
.playbutton {
    width: 60px;
}
- For the search icon, add a position absolute and top and left values.

### once done with the changes, please ask for another code review/

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
**Status: changes required
Hello John,** thank you for implementing the change suggested by the TSE. We are very close to getting approval for this project. We need to remove the horizontal scroll bar. It was introduced due to the  section containing 4 articles. Please fix it and submit for another review.
Screenshot from 2020-06-25 18-14-15

[Optional] when we focus on the search input field. It overflows the navbar. I encourage you to fix that. This is not a hard requirement but nice to have one.

If you need help or have a question regarding the review, DM me on slack **Suman Shrestha
Happy coding**

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Change required :recycle: 

### Hi everyone

### You really need to be careful about force pushing, a previous TSE explain to you that this is a bad practice and can introduce/ broke your page,  this is the page when I opened it in the browser:
Screenshot from 2020-06-25 14-33-49
as you can see it is broke again. Please fix that and don't force push again.

#### Once you are done, ask for another code review

**Comment by mujohn26** [comment]
@HADDADSOHAIB The page really looks good I think the problem might be the browser because on my side it's working perfectly and also when it comes to other TSEs as I see on their screenshot it really looks good using this link https://rawcdn.githack.com/mujohn26/apple-store/feature/design-web/index.html you can check on this screenshot 
Screen Shot 2020-06-25 at 15 53 29

**Comment by HADDADSOHAIB** [comment]
Hi @mujohn26 , after further investigation I found out that there is an issue when I see the page on chrome, when I open the page in other browser (firefox) it looks ok.
Screenshot from 2020-06-25 14-57-09
I am sorry for the inconvenience and I hope that the next reviewer don't face any issues.

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🛑

## Hello @mujohn26 👋. You have done an incredible thus far 👏✨! Thanks a lot for making the changes suggested by the previous reviewers.

### But there are a couple of issues that need to be addressed before we can merge. Check the comments below.

File: index.html
Comment (tirthajyoti-ghosh) [inline index.html]: In-line styles are [bad practice](https://speckyboy.com/good-bad-css-practices). All styles should be in the CSS file.

File: styles/index.css
Comment (tirthajyoti-ghosh) [inline styles/index.css]: Remove `!important` from your CSS file, it's a [bad practice](https://speckyboy.com/good-bad-css-practices). Use more specific selectors instead.

**Review by tirthajyoti-ghosh — COMMENTED** [review]
#### [REQUIRED] Your project headline in README is from a different project. Changes that.
Remember about CSS bad practices. 

#### [OPTIONAL]
- Be sure to add the social handles of the author of the project README.
- The input filed in the navbar is still overflowing when clicked.

Applying the [REQUIRED] changes and consider working on the [OPTIONAL] changes suggested.

Request for another code review when you are done.

Happy coding 👨‍💻

**Comment by mujohn26** [comment]
> #### [REQUIRED] Your project headline in README is from a different project. Changes that.
> Remember about CSS bad practices.
> 
> #### [OPTIONAL]
> * Be sure to add the social handles of the author of the project README.
> * The input filed in the navbar is still overflowing when clicked.
> 
> Applying the [REQUIRED] changes and consider working on the [OPTIONAL] changes suggested.
> 
> Request for another code review when you are done.
> 
> Happy coding 👨‍💻

@tirthajyoti-ghosh I implemented all the changes you requested and also I updated my README

File: styles/index.css
Comment (mujohn26) [reply styles/index.css]: Done

File: index.html
Comment (mujohn26) [reply index.html]: Done

**Review by tirthajyoti-ghosh — APPROVED** [review]
# Status: Approved ✅

## Thanks for implementing the changes. It's a 👍 from me. Go ahead and merge it!

### Onto the next project 🚀 !

Happy coding 👨‍💻

**Pull request merged**
